NPM error "ENOENT: no such file or directory" with source:build:frontend-assets

My system information:

* Operating system type: macOS
* Operating system version: 10.13
* BLT version: 9.2.0-alpha3

Output of `blt doctor`:

```
vagrant@local:/var/www/ptl$ blt doctor
+---------------------------+----------------------------------------------------+
| Property                  | Value                                              |
+---------------------------+----------------------------------------------------+
| %paths.%root              | /var/www/ptl/docroot                               |
| %paths.%site              | sites/default                                      |
| %paths.%modules           | sites/all/modules                                  |
| %paths.%themes            | sites/all/themes                                   |
| %paths.%config-sync       | /var/www/ptl/config/default                        |
| %paths.%files             | sites/default/files                                |
| %paths.%temp              | /tmp                                               |
| %paths.%private           | /var/www/ptl/files-private                         |
| admin-theme               | seven                                              |
| alias-searchpaths.0       | /var/www/ptl/drush/sites                           |
| blt-version               | 9.2.0-alpha3                                       |
| bootstrap                 | Successful                                         |
| composer-version          | Composer version 1.7.2 2018-08-16 16:57:12         |
| config-sync               | /var/www/ptl/config/default                        |
| db-driver                 | mysql                                              |
| db-hostname               | localhost                                          |
| db-name                   | drupal                                             |
| db-password               | drupal                                             |
| db-port                   | 3306                                               |
| db-status                 | Connected                                          |
| db-username               | drupal                                             |
| drupal-settings-file      | sites/default/settings.php                         |
| drupal-version            | 8.6.1                                              |
| drush-alias-files.0       | /var/www/ptl/drush/sites/ptl.site.yml              |
| drush-cache-directory     | /home/vagrant/.drush/cache                         |
| drush-conf.0              | /var/www/ptl/vendor/drush/drush/drush.yml          |
| drush-conf.1              | /var/www/ptl/drush/drush.yml                       |
| drush-conf.2              | /var/www/ptl/docroot/sites/default/local.drush.yml |
| drush-script              | /var/www/ptl/vendor/bin/drush                      |
| drush-temp                | /tmp                                               |
| drush-version             | 9.4.0                                              |
| files                     | sites/default/files                                |
| install-profile           | standard                                           |
| modules                   | sites/all/modules                                  |
| php-bin                   | /usr/bin/php7.1                                    |
| php-conf.0                | /etc/php/7.1/cli/php.ini                           |
| php-os                    | Linux                                              |
| private                   | /var/www/ptl/files-private                         |
| root                      | /var/www/ptl/docroot                               |
| site                      | sites/default                                      |
| stacks.drupal-vm.inited   | true                                               |
| stacks.dev-desktop.inited | false                                              |
| temp                      | /tmp                                               |
| theme                     | ptl_theme                                          |
| themes                    | sites/all/themes                                   |
| uri                       | http://local.ptl.com                               |
+---------------------------+----------------------------------------------------+
+--------------------------------------+--------------------------------------------------------------+
| Check                                | Problem                                                      |
+--------------------------------------+--------------------------------------------------------------+
| ConfigCheck:checkGitConfig           | Git repositories are not defined in blt.yml.                 |
|                                      |   Add values for git.remotes to blt.yml to enabled automated |
|                                      | deployment.                                                  |
| NodeCheck:checkNodeVersionFileExists | Neither .nvmrc nor .node-version file found in repo root.    |
+--------------------------------------+--------------------------------------------------------------+
```

When I run this command:
```
vagrant@local:/var/www/ptl$ blt source:build:frontend-assets
```

I get the following output:
```
Executing frontend-assets target hook...
[ExecStack] npm run build
[ExecStack] Running npm run build in /var/www/ptl/docroot/sites/all/themes/custom/ptl_theme.
npm ERR! Linux 4.4.0-131-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "build"
npm ERR! node v6.14.4
npm ERR! npm  v3.10.10
npm ERR! path /var/www/ptl/package.json
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open

npm ERR! enoent ENOENT: no such file or directory, open '/var/www/ptl/package.json'
npm ERR! enoent ENOENT: no such file or directory, open '/var/www/ptl/package.json'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /var/www/ptl/npm-debug.log
[ExecStack]  Exit code 254  Time 0.975s
[error]  Executing target-hook frontend-assets failed. 
```

And I expected this to happen:

It should run the `npm run build` command as it would run inside the theme directory.

It looks like it's running that command inside the project root directory for some reason?

--- a/config/build.yml
+++ b/config/build.yml
@@ -164,18 +164,18 @@ sync:
 command-hooks:
   # Executed when front end dependencies should be installed.
   frontend-reqs:
-    # E.g., ${docroot}/sites/all/themes/custom/mytheme.
+    # E.g., ${docroot}/themes/custom/mytheme
     dir: ${docroot}
     # E.g., '[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" && nvm use 4.4.4 && npm install'
     command: null
   # Executed when front end assets should be generated.
   frontend-assets:
-    # E.g., ${docroot}/sites/all/themes/custom/mytheme.
+    # E.g., ${docroot}/themes/custom/mytheme
     dir: ${docroot}
     # E.g., '[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" && nvm use 4.4.4 && npm build'
     command: null
   frontend-test:
-    # E.g., ${docroot}/sites/all/themes/custom/mytheme.
+    # E.g., ${docroot}/themes/custom/mytheme
     dir: ${docroot}
     # E.g., `npm test`.
     command: null

--- a/docs/frontend.md
+++ b/docs/frontend.md
@@ -22,7 +22,7 @@ During the execution of `blt setup`, BLT will execute `command-hooks.frontend-re
 
     command-hooks:
       frontend-reqs:
-        dir: ${docroot}/sites/all/themes/custom/mytheme.
+        dir: ${docroot}/themes/custom/mytheme
         command: '[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" && nvm use 4.4.4 && npm install'
 
 If you are using a sub theme of Cog, executing `npm install` in your theme directory (as exemplified above) will install all dependencies listed in [package.json](https://github.com/acquia-pso/cog/blob/8.x-1.x/STARTERKIT/package.json).
@@ -33,7 +33,7 @@ During the execution of `blt setup` and `blt artifact:deploy`, BLT will execute 
 
     command-hooks:
       frontend-assets:
-        dir: ${docroot}/sites/all/themes/custom/mytheme.
+        dir: ${docroot}/themes/custom/mytheme
         command: '[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" && nvm use 4.4.4 && npm run build'
 
 If you are using a sub theme of Cog, executing `npm run build` in your theme directory (as exemplified above) will execute the command defined in `scripts.build` in [package.json](https://github.com/acquia-pso/cog/blob/8.x-1.x/STARTERKIT/package.json#L51).
@@ -44,7 +44,7 @@ During the execution of `blt tests`, BLT will execute `command-hooks.frontend-te
 
     command-hooks:
       frontend-test:
-        dir: ${docroot}/sites/all/themes/custom/mytheme.
+        dir: ${docroot}/themes/custom/mytheme
         command: '[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" && nvm use 4.4.4 && npm test'
 
 If you are using a sub theme of Cog, executing `npm test` in your theme directory (as exemplified above) will execute the command defined in `scripts.test` in [package.json](https://github.com/acquia-pso/cog/blob/8.x-1.x/STARTERKIT/package.json).

